src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Textarea from 'react-textarea-autosize';
+import { PropTypes } from 'prop-types';
+import {
+  handleCommentInput,
+  submitComment,
+  deleteComment,
+  updateComment,
+  setUpdateCommentBody,
+} from '../../redux/actions/commentAction';
+import CommentRender from './CommentRender';
+
+export class Comment extends Component {
+  addComment = () => {
+    const { body, articleSlug, onSubmitComment } = this.props;
+    onSubmitComment(body, articleSlug);
+  }
+
+  onChange = (e) => {
```

Comment:
`( )`

Reply:
I am not sure I understand what you mean by this

Reply:
For `ES6` functions when you have a single argument you can get rid of the brackets. In this case

```
onChange = e => { }
```

Reply:
ohh I see let me handle it